### PR TITLE
Add chemistry param to optimus adapter

### PR DESCRIPTION
**Pull request by samanehsan**
### Purpose
Update the Optimus adapter to work for v1.4.0.

### Changes
- Pass in `chemistry` parameter

### Review Instructions
- No instructions.

### CODE REVIEW

File: pipelines/optimus/adapter.wdl [hunk 1]
```diff
@@ -140,7 +141,7 @@ workflow AdapterOptimus {
   Boolean record_http = false
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.57.0"
+  String pipeline_tools_version = "se-add-optimus-chemistry-param"
```
Comment (rhiananthony) [inline pipelines/optimus/adapter.wdl]: You'll update this to a versioned release, right? :)

**Review by rhiananthony — CHANGES_REQUESTED** [review]
Just the one comment. Everything else looked good to me

File: pipelines/optimus/adapter.wdl (hunk 1 above)
Comment (samanehsan) [reply pipelines/optimus/adapter.wdl]: Yes! This is just for testing :)

Comment (samanehsan) [reply pipelines/optimus/adapter.wdl]: Done making that change @rhiananthony!

**Comment by samanehsan** [comment]
Here is the mintegration test workflow that I ran for Optimus using v1.4.0, this branch of pipeline-tools and the corresponding branch for the adapter-pipeline: https://job-manager.caas-prod.broadinstitute.org/jobs/0555fc3d-23b0-4269-9a52-60a9fa457e36

**Pull request merged**